src/pages/Dashboard/dashboard.tsx

```diff
@@ -0,0 +1,36 @@
+import { useDispatch } from 'react-redux';
+import { revokeCredentials } from 'store/auth.reducer';
+import ButtonLarge, { ButtonLargeVariants } from 'components/shared/ButtonsLarge';
+import { useLogOutMutation } from 'services/blackMarketApi';
+
+const Dashboard = () => {
+  const dispatch = useDispatch();
+
+  const [logOutRequest] = useLogOutMutation();
+
+  async function handleLogout() {
+    const body = {
+      refresh: localStorage.getItem('refresh_token'),
+    };
+    try {
+      await logOutRequest(body);
+    } catch (error) {
+      return;
+    }
+    dispatch(revokeCredentials());
+  }
+
+  return (
+    <div className='absolute m-0 h-screen w-full bg-[url("/src/assets/background.jpg")] bg-cover p-0'>
```

Comment:
WDYT about creating a custom backgroundImage in the tailwind.config.js file instead this arbitrary value